nodeadm/doc/examples.md

```diff
@@ -61,6 +61,38 @@ spec:
 
 The configuration objects will be merged in the order they appear in the MIME multi-part document, meaning the value in the lattermost configuration object will take precedence.
 
+---
+## Configuring node name (experimental)
```

Comment:
Clearer title since we aren't talking about setting the node name to an arbitrary value:
```suggestion
## Using instance ID as node name (experimental)
```